### PR TITLE
Updated README.md with relevant EventStore usefulness/documentation

### PR DESCRIPTION
**Pull request by megan-shah1**
Added more information about EventStoreDB, what it is and its benefits, and how this database differs from a relational database, included use cases regarding how EventStore DB can be used in different industries, updated support and tutorials to provide more sources about event sourcing and overall event driven database creation, added more documentation and included contact information

### CODE REVIEW

**Review by gregoryyoung — COMMENTED** [review]
.

File: README.md [hunk 1]
```diff
@@ -21,10 +42,12 @@ Information on support and commercial tools such as LDAP authentication can be f
 ![Build](https://github.com/EventStore/EventStore/actions/workflows/build-container-focal.yml/badge.svg)
 
 ## Documentation
-Documentation is located in the [`docs`](/docs) folder. It's orchestrated in the separate [documentation repository](https://github.com/EventStore/documentation). It's available online at https://developers.eventstore.com/.
+Documentation is located in the [`docs`](/docs) folder. It's orchestrated in the separate [documentation repository](https://github.com/EventStore/documentation). It's available online at https://developers.eventstore.com/. This documentation guide contains information about the server and clients/APIs.
 
 Read more in the [documentation contribution guidelines](./CONTRIBUTING.md#documentation).
 
+Learn more about event streams here: https://developers.eventstore.com/server/v22.10/streams.html#metadata-and-reserved-names
```
Comment (ylorph) [inline README.md:49]: No sure this is relevant for a readme .

File: README.md [hunk 2]
```diff
@@ -37,6 +60,8 @@ We also host native packages for Linux on [Package Cloud](https://packagecloud.i
 
 EventStoreDB is written in a mixture of C#, C++ and JavaScript. It can run on Windows, Linux and macOS (using Docker) using the .NET Core runtime. However, the projections library (which uses the V8 javascript engine) contains platform specific code and it must be built for the platform on which you intend to run it.
```
Comment (ylorph) [inline README.md:61]: do we still have C++ , that was for the V8 engine no ? We got it out completely in current supported releases

File: README.md [hunk 3]
```diff
@@ -148,3 +173,6 @@ Read more in the [contribution guidelines](./CONTRIBUTING.md).
 ### Proto Changes
 
 If you update the protos, continuous integration will fail. After ensuring the proto change is backwards compatible, please run `./protolock.sh commit` at the root of this repository.
+
+## Contact
```
Comment (ylorph) [inline README.md:177]: It's more for commercial support than any contact no ?

File: README.md [hunk 4]
```diff
@@ -1,13 +1,34 @@
...
... develop code, the code goes through multiple stages in order to optimize efficiency, combat errors, or natural progress as a product becomes more developed. Sometimes, later, some of these changes are rendered unecessary. Relational databases become hard to use in these cases, because they overwrite current data and make it impossible for users to go back to previous versions of their code. Moreover, data needs to be recorded before the changes can be pushed and in some cases, change logs are needed to piece together information. Event Store DB alleviates this problem by using State Transition Databases which log changes automatically and also why the changes were made. This makes accessing and reverting to previosu versions of user code significantly easier. EventStore DB creates a log of these changes and users are able to access virtually any part of the stream in order to see how your data changed and make predictions/projections about future data. Users are also able to track effects of events to see what may have causes changes. Previous data is also immutable but the overall result is scalable and provides client SDKs for major languages like .NET, Java, Go, Node, and Rust.
```
Comment (ylorph) [inline README.md:3]: 🤔 ... some statements are not correct 
Need more time to ponder

**Pull request closed without merging**